detection_rules/etc/version.lock.json

```diff
@@ -2,7 +2,7 @@
   "000047bb-b27a-47ec-8b62-ef1a5d2c9e19": {
     "min_stack_version": "8.3",
     "previous": {
-      "7.16": {
+      "8.3": {
```

Comment:
There may be an existing bug where after we trim this file and run lock versions again, the buffer spaces are then removed and instead the current rule version is 1+ ahead of the fork.

<img width="1714" alt="Screen Shot 2023-01-09 at 2 38 32 PM" src="https://user-images.githubusercontent.com/99630311/211393411-a80c8ca0-887f-4ef3-8c2e-96d4d5ef2f3d.png">